pandas/tests/io/parser/c_parser_only.py

```diff
@@ -496,3 +496,13 @@ def test_bytes_exceed_2gb(self):
             ['x' * (1 << 20) for _ in range(2100)]))
         df = self.read_csv(csv, low_memory=False)
         assert not df.empty
+
+    def test_casting_boolean_nas(self):
+        data = "c1,c2\nfalse,1\n,1"
```

Comment:
Can you add the github issue number here as a comment?

Reply:
Done. Thank you.